scripts/start.sh

```diff
@@ -2,14 +2,11 @@
 
 set -ex
 
-if [ ! -f "./.env" ]; then
-  echo -e "\033[1;31m WARNING: Missing .env file, see CONTRIBUTING.md. \033[0m"
-fi
-
-export $(cat .env | grep NODE_ENV | xargs)
-
-if [ "$NODE_ENV" = "development" ]; then
-  node -r dotenv/config --max_old_space_size=1024 ./src
-else
+if [ "$NODE_ENV" = "production" ]; then
   node --max_old_space_size=1024 ./src
+else
+  if [ ! -f "./.env" ]; then
+    echo -e "\033[1;31m WARNING: Missing .env file, see CONTRIBUTING.md. \033[0m"
+  fi
+  node -r dotenv/config --max_old_space_size=1024 ./src
```

Comment:
Much nicer 👍 Would you mind updating Positron with the same?

Reply:
No problem. Will do. 👍